app/core/Tracker2/pageUnload.js

```diff
@@ -0,0 +1,72 @@
+function getKeyForNamespace (namespace) {
+  return `coco.tracker.pageUnloadRetries.${namespace}`;
+}
+
+function getNamespaceData (namespace) {
+  const namespaceKey = getKeyForNamespace(namespace)
+  const namespaceRetriesJson = window.sessionStorage.getItem(namespaceKey)
+
+  return JSON.parse(namespaceRetriesJson || '[]')
+}
+
+function setNamespaceData(namespace, data) {
+  const namespaceKey = getKeyForNamespace(namespace)
+
+  if (data) {
+    window.sessionStorage.setItem(namespaceKey, JSON.stringify(data))
```

Comment:
Is it possible the user's platform doesn't support sessionStorage?

Reply:
![image](https://user-images.githubusercontent.com/1000442/76559710-e3cf3d80-645c-11ea-8e28-50d3bb79f43d.png)

Think we're good!